mne/tests/test_dipole.py

```diff
@@ -146,7 +146,7 @@ def test_dipole_fitting(tmpdir):
                              subjects_dir=subjects_dir)
     head_to_mni_dip_pos = head_to_mni(dip.pos, 'sample', fwd['mri_head_t'],
                                       subjects_dir=subjects_dir)
-    assert_array_equal(dip_mni_pos, head_to_mni_dip_pos)
+    assert_allclose(dip_mni_pos, head_to_mni_dip_pos, rtol=1e-3, atol=0)
```

Comment:
@larsoner I suggested this change to @vagechirkov because the arrays were coming out as *nearly* identical (all elements different, but only by ~0.00001 mm or so).  But I'm curious why they were different to begin with (presumably `fname_trans` and `fwd['mri_head_t']` are not identical?)  Could this be due to loading one from disk (making it 32-bit precision)?

Reply:
Could be 32-bit precision, or it could be that in one place it's a head->MRI and somewhere it gets converted to MRI->head (or vice-versa) using `linalg.inv`, which can also lose precision (especially if it's after a 32-bit conversion).

Reply:
what do you think is a sensible tolerance here?  since we're talking MNI space (millimeters) I'd be inclined for something like `atol=1e-3, rtol=0` or similar?  that ensures they're within one micrometer of each other

Reply:
That would work, too. The given tolerance is probably also fine given the `rtol=1e-3` and knowledge that the dipole positions are not too wacky.

Reply:
Thank you for the explanation! But I am not sure that I understand the difference between ```atol``` and ```rtol``` in this case.

Reply:
`atol` would allow you to specify absolute-deviation-from-correct, e.g. 0.001 mm or so like @drammock said. `rtol` is relative, so it's like a percentage-from-correct error. In this case it's not going to make a big difference because our values are all within a similar range (probably not off by more than a couple orders of magnitude) and none (presumably) get close to zero.

Reply:
Alright, thank you!